### ISSUE TITLE
Use a simple helper instead of param-case

### ISSUE THREAD

**Issue by ianobermiller** [rofrischmann/inline-style-prefixer#9]
`param-case` and its ilk actually end up being fairly heavyweight (adding about 4k after uglify+gzip). Since you are dealing with simple ascii CSS property names, you should be able to get by with a really simple helper:

```js
const CAMEL_CASE_REGEXP = /([a-z])([A-Z])/g;
const camelToDashCase = name => {
	return name.replace(
		CAMEL_CASE_REGEXP,
		(match, p1, p2) => p1 + '-' + p2.toLowerCase()
	);
};

**Comment by ianobermiller** [comment]
I would have submitted a PR but it appears tests are failing. Also, you have to run `npm run transpile` before running `npm run generate`, so `npm install` was failing.

**Comment by rofrischmann** [comment]
Hey ian! Thanks for that hint, I also thought about param-case being ways to heavy...
And hell I am sorry for this repository being fucked up right now.. I found this package `bowser` which has heavy information on browsers (ways better than this short method I used earlier from stackoverflow), but I actually had no time to fully integrate that. 

I am kind of busy with my college work right now, but starting on wednesday I will again work on this package and fix all issues.
Let me notify you when it is stable again :)
